pipeline/test/services/test-sc.sh

```diff
@@ -7,7 +7,7 @@ if [[ ! -f $1 ]];then
     exit 1
 fi
 export CONFIG_FILE=$1
-
+LOGGING="${2}"
```

Comment:
```suggestion
LOGGING="${2:-}"
```
I assume it is because this is invoked without setting the standard `-euo pipefail` but I think it is good practice to be safe against that. Same in `test-wc.sh`.